lib/buffer.js

```diff
@@ -105,6 +105,10 @@ function fromObject(obj) {
     return b;
   }
 
+  if (obj == null) {
```

Comment:
If we are going to do this, might as well check `!obj`; it's clearer of the intention.